index.html

```diff
@@ -140,11 +140,11 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
                         <div class="input-group">
-                            <input type="text" class="form-control py-3" placeholder="Enter your Email">
+                            <input type="email" class="form-control py-3" placeholder="Enter your Email">
                             <div class="input-group-append">
-                                <div class="input-group-text subscribe-button text-white px-5 font-weight-bold">
+                                <button type="submit" class="input-group-text subscribe-button text-white px-5 font-weight-bold">
```

Comment:
No need of button type submit as there is no form action as of now.

Reply:
done